src/test/java/com/epam/tc/hw2/exercise1/SiteStructureTests.java

```diff
@@ -0,0 +1,200 @@
+package com.epam.tc.hw2.exercise1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchSessionException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class SiteStructureTests {
+    WebDriver webDriver;
+
+    @BeforeClass
+    public void setupClass() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeMethod
+    public void setupTest() {
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    }
+
+    @AfterMethod
+    public void teardown() {
+        if (webDriver != null) {
+            webDriver.quit();
+        }
+    }
+
+    @Test
+    public void siteStructure_isCorrect() {
+
+        SoftAssertions softAssertions = new SoftAssertions();
+
+        // 1. Open test site by URL
+        String siteURL = "https://jdi-testing.github.io/jdi-light/index.html";
+        webDriver.navigate().to(siteURL);
+        String actualURL = webDriver.getCurrentUrl();
+        softAssertions.assertThat(actualURL)
+                      .as("Test site is opened")
+                      .isEqualTo(siteURL);
+
+        // 2. Assert Browser title
+        String title = webDriver.getTitle();
+        softAssertions.assertThat(title)
+                      .as("Browser title equals 'Home Page'")
+                      .isEqualTo("Home Page");
+
+        // 3. Perform login
+        WebElement userIcon = webDriver.findElement(By.id("user-icon"));
+        userIcon.click();
+
+        WebElement userNameInput = webDriver.findElement(By.cssSelector("input#name"));
+        WebElement userPasswordInput = webDriver.findElement(By.cssSelector("input#password"));
+        WebElement loginButton = webDriver.findElement(By.cssSelector("button#login-button"));
+        userNameInput.sendKeys("Roman");
+        userPasswordInput.sendKeys("Jdi1234");
+        loginButton.click();
+        softAssertions.assertThat(loginButton.isDisplayed())
+                      .as("User is logged")
+                      .isEqualTo(false);
```

Comment:
`isFalse()` instead of `isEqualTo(false)`

Reply:
fixed